### DISCUSSION TITLE
華夏動畫

### DISCUSSION THREAD

**Discussion by gigitilto** [1101-GS4719-C/Python-Community-GS4719-C#2338] [General]
從國中的時候開始，我就很喜歡看動畫，各個方面各種類型都略有接觸，也包含中國的動畫，可當我與同學以及教授分享時，他們卻非常的不屑，老實說中國產出一堆3D動畫，3D動畫確實不是每個人都可以接受，但他們也有很多好看的2D動畫，最近中國動畫的品質也越來越高了，我覺得比大部分的日本動漫品質高太多了，他們只是因為動畫是中國的就不看。我覺得不應該因為兩岸的對立而仇恨他們的動畫，我們身邊也有許多中國的製品，既然沒有抵制為何要這樣看不起他們的動畫呢？

**Comment by zoo100130** [comment]
單純就畫面的話，普遍應該還是日本勝，最有名就京阿尼、骨頭社等知名動畫公司
中國動畫我覺得勝點在於劇情，像狐妖小紅娘、刺客五六七、羅小黑戰記、一人之下等我都覺得非常非常不錯(有些我到現在還再追漫畫
但並不是說中國就沒有優質的作畫像悟空、哪吒這些畫面細膩度都無法挑剔，但這些就真的像原po說得一樣大都是3d大作
不得不說中國在動漫界崛起不是一個新聞了，但普遍水平上我還是認為日本在品質、版權把控上是優於中國，看過太多爛尾作品和版權有爭議的(Like仙王日常  btw我還是會看XD)
